apps/webservice/src/app/[workspaceSlug]/(app)/_components/release-condition/useReleaseFilter.ts

```diff
@@ -24,54 +23,33 @@ export const useReleaseFilter = () => {
   );
 
   const setFilter = useCallback(
-    (filter: ReleaseCondition | undefined) => {
+    (filter?: ReleaseCondition, releaseChannelId?: string | null) => {
       if (filter == null) {
         const query = new URLSearchParams(window.location.search);
         query.delete("filter");
+        if (releaseChannelId === null) query.delete("release-channel");
+        if (releaseChannelId != null)
+          query.set("release-channel", releaseChannelId);
```

Comment:
```suggestion
          query.set("release_channel_id", releaseChannelId);
```
following the pattern